homeworks/dankhazipeter/9_fastapi/database.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.ext.asyncio import async_sessionmaker
+from sqlalchemy.ext.asyncio import create_async_engine
+from sqlalchemy.orm import declarative_base
+
+DATABASE_URL = "sqlite+aiosqlite:///./movies.db"
+
+engine = create_async_engine(DATABASE_URL, echo=True)
+
+AsyncSessionLocal = async_sessionmaker(
```

Comment:
Itt meg ezt írta az órai megoldásra: No overloads for "__init__" match the provided argumentsPylance[reportCallIssue](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportCallIssue)